src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_base_layout.html.twig

```diff
@@ -147,10 +147,14 @@
 {% endblock choice_label %}
 
 {% block checkbox_label -%}
+    {%- set label_attr = label_attr|merge({for: id}) -%}
```

Comment:
Missing quotes `{'for': id}`

Reply:
really? I never quote the keys in Twig (unless strictly required because of white spaces, etc.)

Reply:
Quoting is no requirement in Twig. I thought Sebastian suggested it because it is a part of the code standard.